example/src/CSE.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Button, SafeAreaView } from 'react-native';
+import { NativeModules } from 'react-native';
+import { ENVIRONMENT } from './Configuration';
+import Styles from './Styles';
+
+const { AdyenCSE } = NativeModules;
+
+const CseView = () => {
+  const tryEncryptCard = async () => {
+    let unencryptedCard = {
+      number: '5454 5454 5454 5454',
+      expiryMonth: '03',
+      expiryYear: '2030',
+      cvv: '737',
+    };
+    let publicKey = ENVIRONMENT.publicKey;
```

Comment:
Perhaps this publicKey could be moved to the top of the file after the imports, as it is a constant value used in this Component. You don't have to define it locally in both functions

```jsx
...
import { ENVIRONMENT } from './Configuration';
import Styles from './Styles';

const PUBLIC_KEY = ENVIRONMENT.publicKey;
...
```

Reply:
This will probably change (or maybe even removed).
I'll address this in next PR